Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:3.2.2
```

Comment:
There is a direct amd64 variant `amd64/ruby:3.2.2`

Reply:
@akarsh would you be able to do a PR updating it please?

Reply:
Thanks. I'm fine with using the `platform` directive for now.